about.html

```diff
@@ -12,52 +12,88 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
-    <img src="img/about-plan.png" alt="strategy">
+  
+  
+  
+  <div class="container about-page">
     
-    Strategy
+    <header>
+      <div class="top-row">
+         
+          <nav>
+            <img src="img/lambda-black.png">
+              <a href="index.html">Home</a>
+              <a href="about.html">About</a>
+              <a href="#">Products</a>
+              <a href="#">Blog</a>
+              <a href="#">Contact</a>
+          </nav>
+      </div>
+      <img src="img/jumbo-about.png"  class="hdrimg">
+    </header>  
+
+    <div class="about-top-text">
+    <h2>About</h2>
+
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+
+      </div>
+    <section class="atiles">
+      
+      <div class="abox">
+      <img src="img/about-plan.png" alt="strategy">
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+    <h2>Strategy</h2>
+    
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
+    <button name="button">Learn More</button>
+    </div>
 
```

Comment:
For each of these sections, you can reuse a class instead of creating a class for each one.